app/classification.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const teacherArray = ['taught','coached', 'trained', 'educated'];
+const studentArray = ['learned from'];
+const wroteArray = ['wrote','created', 'composed'];
+const playedArray = ['played','performed'];
+
+const dict = {teacher:teacherArray, student:studentArray, wrote:wroteArray, played:playedArray};
+
+module.exports.findRelationshipClass = function(word) {
+  for(let key in dict) {
```

Comment:
we have to consider the case sensitivity here otherwise we might a lot of relationships.  We can use string.toLowerCase() before checking for whether it is present in dict or not!